bluetail/admin.py

```diff
@@ -15,4 +15,4 @@ def get_readonly_fields(self, request, obj=None):
 
 admin.site.register(Flag, FlagAdmin)
 admin.site.register(FlagAttachment)
-admin.site.register(OCDSReleaseJSON)
+admin.site.register(OCDSRecordJSON)
```

Comment:
Any reason why we're removing the `OCDSReleaseJSON`?

Reply:
No reason just that having it there and it being un updatable risks users running into the exceptions we saw earlier.